tests/templates/kuttl/smoke/08-install-hive.yaml.j2

```diff
@@ -23,6 +23,9 @@ spec:
     config:
       logging:
         enableVectorAgent: {{ lookup('env', 'VECTOR_AGGREGATOR') | length > 0 }}
+      resources:
+        memory:
+          limit: "1Gi"
```

Comment:
@sbernauer, ideally the operator should set this value to something higher to avoid OOMKilled:

```
    State:          Waiting
      Reason:       CrashLoopBackOff
    Last State:     Terminated
      Reason:       OOMKilled
      Exit Code:    137
      Started:      Mon, 24 Feb 2025 10:22:26 +0100
      Finished:     Mon, 24 Feb 2025 10:22:50 +0100
    Ready:          False
    Restart Count:  3
    Limits:
      cpu:     1
      memory:  512Mi
    Requests:
      cpu:      250m
      memory:   512Mi
```

Reply:
Big question is if this is already happening with hive-op 24.11.1 ;)
Might be the case it was always this way

Reply:
It hadn't been failing that way until a recent rebase/merge commit.

Reply:
Hmm I guess we should upper the hive-metastore default resources in case 512 MB are not enough the run trino smoke tests ...
WDYT?

Reply:
Default updated in https://github.com/stackabletech/hive-operator/pull/578

Reply:
Resource overrides reverted in fe50cc7d29721471bb642f9a7a08baaa8370d957

Reply:
Thanks for upping the defaults!
We should now IMHO remove the explicit resources here

Reply:
Beat me to it :D